### DISCUSSION TITLE
OneDrive Colorful Output

### DISCUSSION THREAD

**Discussion by TheBusyDev** [abraunegg/onedrive#2204] [Ideas]
Hi, I use your OneDrive Client for my daily work and I really appreciate it! 
I had a look to your [Colorful log output terminal modification](https://github.com/zzzdeb/dotfiles/blob/master/scripts/tools/onedrive_log) in your [README.md page](https://github.com/abraunegg/onedrive#external-enhancements).
As I found this script really useful, I improved it and made my own version: [https://github.com/TheBusyDev/onedrive_2.0](https://github.com/TheBusyDev/onedrive_2.0). 
It may be useful even to you... Bye!!

**Comment by abraunegg** [comment]
@TheBusyDev 
Thanks for your contribution to the community

**Reply by TheBusyDev** [reply]
By the way, if you think this script can be helpful to others, could you add it to the [External Enhancements](https://github.com/abraunegg/onedrive#external-enhancements) section please? 🙏
